apps/nova/src/components/TutorProfile/Ratings.tsx

```diff
@@ -1,74 +1,107 @@
-import { Loading } from "@litespace/luna/Loading";
-import { Timeline, TimelineItem } from "@litespace/luna/Timeline";
-import { IRating } from "@litespace/types";
-import { UseQueryResult } from "@tanstack/react-query";
-import React, { useMemo, useState, useCallback } from "react";
-import { Star } from "react-feather";
-import Title from "@/components/TutorProfile/Title";
-import Rating from "@/components/TutorProfile/Rating";
-import EditRating from "@/components/TutorProfile/EditRating";
-import DeleteRating from "@/components/TutorProfile/DeleteRating";
+import { useFindTutorRatings } from "@litespace/headless/rating";
+import React, { useCallback } from "react";
+import { LoadingTutorInfo } from "@/components/TutorProfile/LoadingTutorInfo";
+import { ratingOrganizer } from "@/lib/ratings";
+import { useUser } from "@litespace/headless/context/user";
+import {
+  TutorRatingCard,
+  TutorRatingCardGroup,
+} from "@litespace/luna/TutorFeedback";
+import { Typography } from "@litespace/luna/Typography";
+import { useFormatMessage } from "@litespace/luna/hooks/intl";
+import Star from "@litespace/assets/Star";
+import NewTutor from "@litespace/assets/NewTutor";
+import { Button, ButtonSize } from "@litespace/luna/Button";
+import { isEmpty } from "lodash";
+import { TutorLoadingError } from "@/components/TutorProfile/TutorLoadingError";
 
-const Ratings: React.FC<{
-  query: UseQueryResult<IRating.FindRateeRatingsApiResponse, Error>;
-}> = ({ query }) => {
-  const [editRating, setEditRating] = useState<IRating.Populated | null>(null);
-  const [deleteRating, setDeleteRating] = useState<{
-    ratingId: number;
-    tutorId: number;
-  } | null>(null);
-
-  const onEditRating = useCallback(
-    (rating: IRating.Populated) => setEditRating(rating),
-    []
+const NoTutorRatings: React.FC<{ tutorName: string | null }> = ({
+  tutorName,
+}) => {
+  const intl = useFormatMessage();
+  return (
+    <div className="flex flex-col mt-20 gap-14 items-center justify-center">
+      <NewTutor />
+      <Typography
+        element="h4"
+        weight="bold"
+        className="text-natural-950 text-center mb-20"
```

Comment:
It works I know but this is not the right place to put `mb-20`. I will let you decide where to put it.